src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SpecialMercuryPageObject.java

```diff
@@ -0,0 +1,27 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.mercury;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mobile.MobileBasePageObject;
+
+/**
+ * Created by Lukasz N. on 2014-11-20.
+ */
+public class SpecialMercuryPageObject extends MobileBasePageObject {
```

Comment:
This is a good idea to store special pages in packages but it is also a convention change, because so far we stored special pages in package specialPages. But i think it's better for the class to be 'close' to other project related classes, as it is here.